lib/fast_jsonapi/fieldset.rb

```diff
@@ -0,0 +1,7 @@
+module FastJsonapi
+  class Fieldset
+    def initialize(fields)
+      @fields = fields
```

Comment:
could we run define and run deep_symbolize method here? I think it makes sense for it to be transparent to the rest of the code.

Reply:
That's a remnant of a previous attempt and no longer needed. I'll delete it and remove the reference